util/exec.go

```diff
@@ -23,6 +23,7 @@ import (
 // from the binary's stdout and stderr to this program's stdout and stderr.
 func Exec(bin string, args ...string) error {
 	cmd := exec.Command(bin, args...)
```

Comment:
Consider using https://golang.org/pkg/os/exec/#LookPath as a nicety here (and returning a friendly error) instead of just going straight to the exec